go.mod

```diff
@@ -64,3 +58,75 @@ require (
 	k8s.io/apimachinery v0.21.2
 	k8s.io/client-go v0.21.2
 )
+
+require (
```

Comment:
Why this change?

Reply:
I'm assuming it's the result of running `go mod tidy`?

Reply:
Could be. But if not needed we could as well as get rid of some of these dependencies.

Reply:
It is due to changes in behavior in Go regarding module dependency tracking in Go 1.17. See 'Pruned module graphs in go 1.17 modules' in https://go.dev/doc/go1.17

Reply:
Thank you.